bpf/lib/egress_policies.h

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/* Copyright (C) 2021 Authors of Cilium */
+
+#ifndef __LIB_EGRESS_POLICIES_H_
+#define __LIB_EGRESS_POLICIES_H_
+
+#ifdef ENABLE_EGRESS_GATEWAY
+/* is_egress_destination returns true if the given destination is part of the
```

Comment:
Seems like the function name was renamed, but not the comment.

```suggestion
/* is_cluster_destination returns true if the given destination is part of the
```

nit: would `is_cluster_local` be a better name?

Reply:
Nice catch, thanks!